psydac/linalg/null.py

```diff
@@ -34,6 +34,12 @@ def codomain( self ):
     def dtype( self ):
         return self.domain.dtype
 
+    def toarray(self):
+        raise NotImplementedError('toarray() is not defined for this outdated class.')
+
+    def tosparse(self):
+        raise NotImplementedError('tosparse() is not defined for this outdated class.')
+
```

Comment:
What do you mean with "outdated class"? Is this not used anymore?

Reply:
Unlike identity.py, null.py is not being used at all.

Reply:
If it's not used I suggest to remove it altogether. I can take care of it

Reply:
Actually, the files `null.py` and `identity.py` were created by @davschneller 1.5 years ago, so we should probably remove both at the same time.

Reply:
I suggest to leave this to a separate PR.